inlong-manager/manager-dao/src/main/resources/mappers/StreamSourceEntityMapper.xml

```diff
@@ -319,7 +319,24 @@
             for update
         </where>
     </select>
-
+    <select id="selectByClusterAndDataNode" resultType="org.apache.inlong.manager.dao.entity.StreamSourceEntity">
+        select
+        <include refid="Base_Column_List"/>
+        from stream_source
+        <where>
+            is_deleted = 0
+            and status not in (99, 110)
```

Comment:
Why filter those status?
Can we description it on the method name, such as `selectNormalXxx` or `selectUnDeletedXxx`.

Reply:
done